emf-builtins/.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file to be committed? If so, could we add a line in the README of this project about this file?

Reply:
This is part of the Eclipse project. As we need Eclipse to sensibly edit the `.ecore` and `.genmodel` files, I'd like to have a proper Eclipse project.

Reply:
Ok, makes sense